### ISSUE TITLE
There is an issue with the circle mask it appears on dark mode

### ISSUE THREAD

**Issue by alexshi0000** [JoseRFelix/react-toggle-dark-mode#40]
When moonColor is changed, you see another shade of color that is present 
![image](https://user-images.githubusercontent.com/19629524/212577483-f7ff376d-0b4f-42c8-83b1-61c21c26f55e.png)

**Comment by chrisfrantz** [comment]
Yeah you need to be able to control the strokeWidth property which means you'll need to fork it to make the change afaik
